pom.xml

```diff
@@ -1,4 +1,8 @@
-<project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
we've been back and forth on whether we want/need these attributes. Not all eclipse suggestions are necessarily good ideas.